src/helper.js

```diff
@@ -100,7 +101,12 @@ export const errorHandler = (error) => {
 };
 
 export const getApiUrl = (asset) => {
-    return pairs[`${asset}/BTC`].apiUrl;
+    const pair = pairs[`${asset}/BTC`];
+    if (!pair) {
+        log.error("no pair found for asset", asset, "using", BTC);
+        return pairs[`${BTC}/BTC`].apiUrl;
+    }
```

Comment:
This is a bad idea. We should overwrite the asset with `setAsset`, not put a hack in this function